go/test/endtoend/vtgate/main_test.go

```diff
@@ -68,10 +68,23 @@ create table t2_id4_idx(
 	primary key(id),
 	key idx_id4(id4)
 ) Engine=InnoDB;
-`
+
+create table t3(
+	id5 bigint,
+	id6 bigint,
+	id7 bigint,
+	primary key(id5)
+) Engine=InnoDB;
+
+create table t3_id7_idx(
+    id bigint not null auto_increment,
+	id7 bigint,
+	id6 bigint,
+    primary key(id)
+) Engine=InnoDB;`
```

Comment:
Although this is not a recommended way to use a lookup vindex, you do show how the feature will behave in such cases. So, I'll allow it.